### PR TITLE
create product card

### PR DESCRIPTION
**Pull request by kyatsenko**
[DEMO LINK](https://kyatsenko.github.io/layout_product-cards/)
[TEST REPORT LINK](https://kyatsenko.github.io/layout_product-cards/report/html_report/)

The second test fails because it does not find the reference.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <div class="card" data-qa="card">
+
+      <img
```
Comment (jstmpelowycz) [inline src/index.html:17]: no spaces between non-siblings
```suggestion
  <body>
    <div class="card" data-qa="card">
      <img
```

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,60 @@
...
+
+      <h2 class="card__title">
+        APPLE A1419 iMac 27&quot Retina 5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <div class="card__product">Product code: 195434</div>
+
+      <div class="card__rating-review">
+
+        <div class="star card__rating">
```
Comment (jstmpelowycz) [inline src/index.html:31]: > no spaces between non-siblings

same here

Comment (jstmpelowycz) [inline src/index.html:31]: seems like you don't use `star` class, so you can remove it

File: src/index.html [hunk 3]
```diff
@@ -1,13 +1,60 @@
...
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+        </div>
+
+
+        <div class="card__review">Reviews: 5</div>
+
+      </div>
```
Comment (jstmpelowycz) [inline src/index.html:44]: > no spaces between non-siblings
```suggestion
        </div>

        <div class="card__review">Reviews: 5</div>
      </div>
```

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,60 @@
...
+
+      <a
+        class="card__button"
+        href="#"
+        data-qa="hover">
+        Buy
+      </a>
+
+    </div>
```
Comment (jstmpelowycz) [inline src/index.html:58]: 2 points: proper formatting + no spaces between non-siblings
```suggestion
      <a
        class="card__button"
        href="#"
        data-qa="hover"
      >
        Buy
      </a>
    </div>
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+/* #region reset */
+body,
+p {
+  margin: 0;
+  padding: 0;
+}
+/* #endregion */
+
+/* #region card */
+.card {
```
Comment (jstmpelowycz) [inline src/style.css:27]: this particularly applies to it — the checklist, p. 11
let's split this styles into dedicated files

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+
+.card__rating-review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+/* #region stars */
+.stars {
```
Comment (jstmpelowycz) [inline src/style.css:66]: the checklist, p.11

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
```
Comment (jstmpelowycz) [inline src/style.css:117]: the checklist, p.8

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
+  background: var(--blue-color);
+  border: 1px solid var(--blue-color);
```
Comment (jstmpelowycz) [inline src/style.css:123]: initially, should be transparent

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,134 @@
...
+.card__price-value {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (jstmpelowycz) [inline src/style.css:116]: flexbox is not for this purpose, it's useful when we have multiple items to arrange

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,134 @@
...
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```
Comment (jstmpelowycz) [inline src/style.css:44]: you already normalize it, don't you?

File: src/style.css [hunk 11]
```diff
@@ -0,0 +1,134 @@
...
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__title {
+  margin: 0 0 4px;
```
Comment (jstmpelowycz) [inline src/style.css:43]: let's reset margins above

File: src/style.css [hunk 12]
```diff
@@ -0,0 +1,134 @@
...
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__image {
+  display: block;
```
Comment (jstmpelowycz) [inline src/style.css:36]: do you need it?

File: src/style.css [hunk 13]
```diff
@@ -0,0 +1,134 @@
...
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+/* #region stars */
+.stars {
+  display: flex;
+  flex-direction: row;
```
Comment (jstmpelowycz) [inline src/style.css:68]: it's the default value

File: src/style.css [hunk 14]
```diff
@@ -0,0 +1,134 @@
+/* #region Normalize */
+* {
+  box-sizing: border-box;
+}
+
+html {
+  --blue-color: #00acdc;
+  --text-dark-color: #060b35;
+  --text-light-color: #616070;
```
Comment (jstmpelowycz) [inline src/style.css:9]: I'd put it to `:root`

**Review by jstmpelowycz — CHANGES_REQUESTED** [review]
awesome, almost done! let's take care of your code, see the comments below 👇

File: src/index.html [hunk 15]
```diff
@@ -1,13 +1,55 @@
...
+        <div class="card__price-value">$2,199</div>
+      </div>
+
+      <a
+        class="card__button"
+        href="#"
+        data-qa="hover"
+        >
+        Buy
+      </a>
```
Comment (SerhiiKirik) [inline src/index.html:52]:
```suggestion
      <a
        class="card__button"
        href="#"
        data-qa="hover"
      >
        Buy
      </a>
```

File: src/card.css [hunk 16]
```diff
@@ -0,0 +1,79 @@
...
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: bold;
+}
+
+.card__button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 11px 0;
```
Comment (SerhiiKirik) [inline src/card.css:63]: Organize your styles 
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)

File: src/card.css [hunk 17]
```diff
@@ -0,0 +1,79 @@
...
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  color: #fff;
```
Comment (SerhiiKirik) [inline src/card.css:67]: Create variables for all colors)
```suggestion
  color: var(--color-white);
```

**Review by SerhiiKirik — APPROVED** [review]
Well done 🔥 
Let's check the comments & move on 👩‍💻

**Pull request closed without merging**